src/resmom/linux/alps.c

```diff
@@ -7298,17 +7311,7 @@ node_group_start(ud_t *d, const XML_Char *el, const XML_Char **atts)
 			else
 				strcpy(node_group->state, BASIL_VAL_UNKNOWN);
 		} else if (strcmp(BASIL_ATR_SPEED, *np) == 0) {
-			if (*node_group->speed != '\0') {
-				parse_err_multiple_attrs(d, *np);
-				return;
-			}
-
-			/* *vp cannot be empty (""), "0" nor negative. */
-			if (strtol(*vp, &invalid_char_ptr, 10) <= 0) {
-				parse_err_illegal_attr_val(d, *np, *vp);
-				return;
-			}
-			strncpy(node_group->speed, *vp, sizeof(node_group->speed) - 1);
+			continue;
```

Comment:
That will not be necessary, since expat parsing will fail if speed attribute appear's twice. Since xml doesn't allow same attribute to  be repeated twice in an element.